src/config.ts

```diff
@@ -78,9 +81,20 @@ export const SCENE_TILE_PROVIDERS = [
     maxZoom:  13,
     name: 'PlanetScope',
     prefix: 'planetscope',
+    catalogSection: 'planet', // Look in /planet/* in the catalog
     provider: 'Planet',
     url: `https://tiles{0-3}.${PLANET_BASE_URL}/v1/experimental/tiles/PSOrthoTile/__SCENE_ID__/{z}/{x}/{y}.png?api_key=__API_KEY__`,
   },
+  {
+    attributions: '<a href="https://docs.opendata.aws/landsat-pds/readme.html">Landsat PDS</a>',
+    maxZoom:  13,
+    name: 'Landsat8 (Public Data Set)',
+    prefix: 'landsat_localindex',
```

Comment:
I believe this is supposed to be landsat_pds if i'm reading the api PR correctly

Reply:
Fixed.